hs_tracking/middleware.py

```diff
@@ -69,6 +72,9 @@ def process_response(self, request, response):
         if not hasattr(request, 'user'):
             return response
 
+        # TODO: #4733
```

Comment:
I did just some rough tests on my development rig, seeing how much this middleware slows down requests.
Here are the results:

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/dcowan/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/dcowan/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
</head>

<body link="#0563C1" vlink="#954F72">



Task | Middleware   delay (ms)
-- | --
DEVIN resource create | 235.1298332
DEVIN file   upload | 100.1884937
DEVIN file   download | 30.07245064
DEVIN bagit   download | 122.7080822
DEVIN logout | 67.07572937
my res page | 105.5552959
signin | 97.81479836



</body>

</html>

I think it is poor form that we are "logging" this metrics/tracking info into the db in the first place.  We have an ELK stack that can import raw logs and index them and provide nice visualization. I think we should be writing this info to logs (ideally asynchronously) instead of storing it in our hydroshare db, only to export it to log files, and then re-ingest into our ELK stack.

Unless HS uses this tracking info somewhere internally in the django app. But it don't think it does.